articles/deploying-your-android-app-to-google-play-store/index.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/deploying-your-android-app-to-google-play-store/index.md
+title: Deploying your Android App to Play Store
+description: This tutorial provides a detailed guide on the steps you should follow when deploying your Android application to the Play Store.
+author: Christine Wasike
+date: 2021-01-20T00:00:00-16:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+  - url: /engineering-education/deploying-your-android-app-to-google-play-store/hero.jpg
+    alt: Google Play Store
+---
+It is always challenging for beginners to deploy their first Android application on the Play Store. There are just too many rules and steps that need to be followed. Failure to adhere to these regulations may impact your application negatively. This tutorial helps you avoid the headaches and negative experiences associated with the app deployment process.
```

Comment:
Can this also reach _often challenging_ instead of _always challenging_ ?
I think that's better language that eludes some sense of hope compared to always facing difficulty.